config.mak.uname

```diff
@@ -718,19 +710,13 @@ else
 		HAVE_LIBCHARSET_H = YesPlease
 		USE_GETTEXT_SCHEME = fallthrough
 		USE_LIBPCRE = YesPlease
-		USE_NED_ALLOCATOR = YesPlease
```

Comment:
Here, we now silently remove the last remaining `USE_NED_ALLOCATOR`, which changes the behavior in an undesired way. Let's just not.